src/Utilities/CssConcatenator/RuleBlock.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * Implementation of media rules sub-property for \Pelago\Emogrifier\Utilities\CssConcatenator
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
+ */
+class RuleBlock
+{
+    /**
+     * Array whose keys are selectors for the rule block (values are of no
+     *     significance);
+     *
+     * @var mixed[]
+     *
+     * @psalm-var array<string, mixed>
+     */
+    public $selectorsAsKeys = [];
```

Comment:
possible `@todo` docblock tags

* review whether one should use `@readonly` when `vimeo/psalm` is upgraded as part of #793

Reply:
> 
> 
> at least one of them is a zero.

Sharp spot.  Presumably because of a combination of `array_flip` and `$x[] =` pushing to the array.  I've not looked closely at the code right now.